### PR TITLE
modified main.js with toggle functionality and added basic styling in…

### PR DESCRIPTION
**Pull request by Doreen2002**
… style.css

### CODE REVIEW

**Comment by Doreen2002** [comment]
***“This is my first PR in the ACA program”._**

File: JS/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const toggle =()=>
```
Comment (Montso-Matlaletsa) [inline JS/main.js:1]: I like that you adopted the new way by using arrow function

File: index.html [hunk 2]
```diff
@@ -10,4 +10,8 @@
 </head>
 <body>
     <p>i love aca.</p>
-</body>
\ No newline at end of file
+    <button onclick="toggle()" class="toggleButton" >Click</button>
+    <div id="toggledText">ACA just taught me about continuous integration and continuous deployment.</div>
+
```
Comment (Montso-Matlaletsa) [inline index.html:15]: try not to use double slashes when navigating to a file as that could cause some problems (on the head section)

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+*
+{
```
Comment (Bandla1995) [inline CSS/style.css:2]: *{ }, this is the syntax i remember, using this one I have never seen this before.

File: CSS/style.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.toggleButton
+{
+    background-color: rgb(128, 245, 245);
+    color: rgb(104, 97, 97);
+    border: solid 1px silver;
+    width: 100px;
+
```
Comment (Bandla1995) [inline CSS/style.css:14]: space left in between, make the code cleaner

File: CSS/style.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+}
+.toggleButton:hover
+{
+   background-color: rgb(250, 250, 244);
+}
+.toggleButton:active
+{
+    border: red 1px solid;
+}
+#toggledText
```
Comment (Bandla1995) [inline CSS/style.css:26]: i like how , you used both div and id observing the #param on this line.

**Review by Bandla1995 — COMMENTED** [review]
good project, just a few mistakes that are manageable

**Pull request merged**